index.qmd

```diff
@@ -0,0 +1,1154 @@
+---
+title: "Project oriented workflows"
+author: "Max Kuhn, E. David Aja, Shannon Pileggi"
+footer:  "[Project oriented workflows]()"
+logo: "img/wtf-logo-square-transparent.png"
+format: 
+  revealjs: 
+    height: 900
+    width: 1600
+    theme: [night, slides.scss]
+    highlight-style: a11y
+    transition: fade
+    slide-number: true
+editor: visual
+execute:
+  freeze: auto
+---
+
+# Getting started
+
+```{r}
+library(countdown)
+```
+
+```{r font-awesome-color}
+# fill for font awesome icons
+fa_fill <- "#C7B41D"
+```
+
+## Licensing
+
+<br>
+
+This work is licensed under a [Creative Commons Attribution-ShareAlike 4.0 International License](https://creativecommons.org/licenses/by-sa/4.0/) (CC BY-SA4.0).
+
+<br>
+
+  - Part 1: Original sides were authored by [**Shannon Pileggi**](https://www.pipinghotdata.com/){target="_blank"}, some added or altered by [**Max Kuhn**](https://max-kuhn.org/){target="_blank"}
```

Comment:
I understand how CC works but I'd like to acknowledge the provenance of the materials. Please edit this if there is a better way to describe what was done before (and by who).